internal/cmd/operator-sdk/generate/packagemanifests/packagemanifests.go

```diff
@@ -35,8 +35,15 @@ towards using bundles by default. Run 'operator-sdk generate bundle -h' for more
 
 Running 'generate packagemanifests' is the first step to publishing your operator to a catalog and/or deploying
 it with OLM. This command generates a set of manifests in a versioned directory and a package manifest file for
-your operator. Typically one would run 'generate kustomize manifests' first to (re)generate kustomize bases
-consumed by this command.
+your operator. A ClusterServiceVersion manifest will be generated from the set of manifests passed to this command
+(see below) using either an existing base at '<kustomize-dir>/bases/<package-name>.clusterserviceversion.yaml',
+typically containing metadata added by 'generate kustomize manifests' or by hand, or from scratch if that base
+does not exist. All non-metadata values in a base will be overwritten.
+
+There are two ways to pass cluster-ready manifests to this command: stdin via a Unix pipe,
+or in a directory using '--input-dir'. See command help for more information on these modes.
+Passing a directory is useful for running this command outside of a project, as kustomize
```

Comment:
I think we need to clarify as first-class its usage for SDK projects and then for running outside of the dir or for Non-SDK projects with examples

Reply:
The examples make both use cases pretty clear IMO.